spec/spec_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'ledger_sync'
```

Comment:
`ledger_sync-xero`

And we should figure out what we need to share from Core here.